app/controllers/likes_controller.rb

```diff
@@ -0,0 +1,13 @@
+class LikesController < ApplicationController
+  before_action :current_user, only: [:create]
+
+  def create
+    @post = Post.find(params[:post_id])
+    new_like = current_user.likes.new(
+      users_id: current_user.id,
+      post_id: @post.id
+    )
```

Comment:
- [ ] <b>REQUIRED:</b>Also, here. When you say `current_user.likes.new()` the user_id is already initialized, so you don't need to re-initialize it again as you did on line 7. meaning your code would simply be like this:

```rb
@post = Post.find(params[:post_id])
@like = current_user.likes.new(post_id: @post.id)
```